### PR TITLE
add format checker & format cmd to github actions

### PR DESCRIPTION
**Pull request by exaexa**
blatant copy from COBREXA

wannabe usage demo here https://github.com/exaexa/ci-format-test/pull/1

cobrexa PR with actual description & pictures here https://github.com/LCSB-BioCore/COBREXA.jl/pull/615

### CODE REVIEW

File: .github/workflows/pr-cmd-format.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+name: PR auto-formatting command
+
+jobs:
+  format:
+    if: ${{ github.event.issue.pull_request && (github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER' || github.event.issue.user.id == github.event.comment.user.id) && startsWith(github.event.comment.body, '/format') }}
+    name: auto-format
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the repository
+        uses: actions/checkout@v2
```
Comment (giordano) [inline .github/workflows/pr-cmd-format.yml:15]: v3 is the latest version now
```suggestion
        uses: actions/checkout@v3
```

**Comment by exaexa** [comment]
> green_circle  Commit [a08c3d7](https://github.com/LCSB-BioCore/SBML.jl/commit/a08c3d7ce5007a81b402c698a220f7330093b27e) is formatted properly.

Thank you github.

**Review by giordano — APPROVED** [review]
Looks great overall!  My only question is: does it comment after every push?  I'm pushing a lot in #143, that'd be _a lot_ of noise :sweat_smile:  Maybe it can comment only if there is something wrong?

**Comment by exaexa** [comment]
> Looks great overall! My only question is: does it comment after every push? I'm pushing a lot in https://github.com/LCSB-BioCore/SBML.jl/pull/143, that'd be a lot of noise sweat_smile Maybe it can comment only if there is something wrong?

Yeah that was a bit of a design decision so that people are encouraged to push once and for all and get the nice green-shining badge of good format, instead of hard-failing the test. Alternative ideas welcome.

Maybe we could delete the previous formatting comment (like codecov does) ? (ofc if there's a howto for that somewhere :D )

**Pull request merged**